Reintroduce the isLast variable

shaavan opened the pull request:
fixes #141 

- This PR reintroduces the isLast variable to keep track of the last challenge and to fix the `Reference Error: Can't find variable: isLast"`

ecurrencyhodler commented:
Thanks for the quick fix! @vinayaksh42 Can you review?

afterburn commented on components/Navbar/Tab.tsx:
```diff
@@ -32,6 +32,7 @@ export default function Tab({
 
   const challengeId = isRouteLesson ? pathData.pop().split('-')[0] : undefined
   const isActive = challenge.lessonId === challengeId
+  const isLast = index == count - 1
```
Let's use `===` to make sure JS doesn't make any assumptions for us. Should work like this, but its a good practice to use `===` whenever you can.

shaavan replied:
Sure! Let me fix this. Always avoid assumptions! ✌️

afterburn reviewed (changes requested):
Left a comment for a small improvement

vinayaksh42 commented:
> Thanks for the quick fix! @vinayaksh42 Can you review?

LGTM
Let's implement Kevin's suggestion for added improvement

vinayaksh42 reviewed (commented):
LGTM
Let's implement Kevin's suggestion for added improvement

shaavan commented:
Updated from ffab86197be36b02ad59289e7dd4262903881ebd to 3ba01cc2905cfe4bc61088ce74e9cd36fbe667a7 ([pr142.01](https://github.com/shaavan/saving-satoshi/commits/pr142.01) -> [pr142.02](https://github.com/shaavan/saving-satoshi/commits/pr142.02), [diff](https://github.com/shaavan/saving-satoshi/compare/pr142.01..pr142.02))
Addressed @afterburn comment.

Change:

- Replace `==` with `===`.
